mimic/rest/cinder_api.py

```diff
@@ -31,7 +31,14 @@ def catalog_entries(self, tenant_id):
         """
         return [
             Entry(
-                tenant_id, "volume", "cloudBlockStorage",
```

Comment:
I don't think there's any reason we couldn't have it registered as both; then we can cover both use-cases.

As to API completeness...well, that's a whole other matter. It basically comes down to what people need and are willing to contribute. This project, AFAICT, is entirely volunteer based.